### ISSUE TITLE
Unable to move after installing the addon.

### ISSUE THREAD

**Issue by ScottVordem** [relaxtakenotes/better-movement-v2#5]
Whenever I enable the addon (via sv_bm_enabled), I cannot move at all except look around and go in noclip. Jumping does not even work. Disabling the addon in this state (via sv_bm_enabled) makes me run at the speed of sound, if not a little faster.

I have no other addons that modify player speeds aside from this one, so I don't know what is wrong. There are no LUA errors either. and none of the modifiers for speed are set to zero. they are the default values you set.

The only way to fix is to completely disable the addon via it's settings and restart the map. in which case re-enabling the addon will break movement again.

I'm on x64 branch if that matters.

**Comment by relaxtakenotes** [comment]
doesnt happen to me under the same circumstances. make sure ur convars are set to their default values

**Comment by ScottVordem** [comment]
Weird, setting them back to the default values via console and restarting the map seemed to have fixed it and changing them afterward stays fixed.....I don't know what happened.

I did noticed before I set them back to default was that some values had a bunch of zeros at the end.

The "inside checks time" was set to 0.800000 instead of 0.8 which I thought was strange.

There were other convars set similarly also, and I think that's what broke it before, but I don't know how they got randomly set like that, perhaps the slider in the spawnmenu settings occasionally glitch? I really don't know.
